runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -241,14 +254,19 @@ private void insertMenuEntry(MenuEntry newEntry, MenuEntry[] entries)
 	}
 
 	private void lookupPlayer(String playerName)
```

Comment:
This is only use one other place, might aswell remove it and use the one which takes the endpoint.